packages/core/components/FileMetadataSearchBar/test/FileMetadataSearchBar.test.tsx

```diff
@@ -0,0 +1,99 @@
+import { configureMockStore } from "@aics/redux-utils";
+import { fireEvent, render } from "@testing-library/react";
+import { expect } from "chai";
+import * as React from "react";
+import { Provider } from "react-redux";
+
+import FileMetadataSearchBar, { DATE_RANGE_SEPARATOR } from "../";
+import FileFilter from "../../../entity/FileFilter";
+import { AnnotationName } from "../../../constants";
+import { initialState, selection } from "../../../state";
+
+describe("<FileMetadataSearchBar />", () => {
+    // TODO: Figure out a way to test this. it doesn't seem like the <Searchbox />
+    // component recognizes the react-testing-library "onSubmit" event signals nor
+    // does it recognize an "Enter" keydown or keypress event.
```

Comment:
Do any of the suggestions here help?

https://stackoverflow.com/questions/62542988/how-to-test-fluent-ui-dropdown-with-react-testing-library

Reply:
I'll try this out!

Reply:
It isn't a "click" per se that this is failing to register. The imported component "Searchbox" appears to listen to "onSubmit" events which ideally for us are triggered via "Enter" keypresses. However, I can't get any of the solutions I've seen thus far to successfully trigger the callback (perhaps there is another issue I'm missing though!).

Reply:
Are you planning to resolve this TODO before merge?